app/src/components/Project/Content/AllCardsContent.tsx

```diff
@@ -101,7 +101,7 @@ export default function AllCardsContent({
                 <RiArrowLeftSLine onClick={handleScrollForward} />
               </div>
             </div>
-            <div ref={scrollContainerRef} className='flex overflow-x-scroll snap-x snap-mandatory'>
+            <div ref={scrollContainerRef} className='flex overflow-x-scroll snap-x snap-mandatory snap-center'>
```

Comment:
not here - to children. Remain the snap-x and snap-mandatory at parent + add snap-center to children
https://tailwindcss.com/docs/scroll-snap-type

Reply:
sorry eugeek I updated it but didn't resolve the problem